README.md

```diff
@@ -42,6 +43,11 @@ FaceChain is powered by [ModelScope](https://github.com/modelscope/modelscope).
 
 
 # Features
+- Support a series of new tricks to stableize Lora training
```

Comment:
stablize

Introduce beta version of Inpaint, with techniques such as in-trianing validation and post-training ensemble to stablize LoRA.